kura/org.eclipse.kura.asset.cloudlet.provider/OSGI-INF/AssetCloudletComponent.xml

```diff
@@ -1,16 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
-    Copyright (c) 2016 Eurotech and/or its affiliates and others
+    Copyright (c) 2016, 2017 Eurotech and/or its affiliates and others
 
      All rights reserved. This program and the accompanying materials
      are made available under the terms of the Eclipse Public License v1.0
      which accompanies this distribution, and is available at
      http://www.eclipse.org/legal/epl-v10.html
 -->
-<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" 
-	activate="activate" 
-	deactivate="deactivate" 
-	name="org.eclipse.kura.asset.cloudlet">
+<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" activate="activate" deactivate="deactivate" immediate="true" name="org.eclipse.kura.asset.cloudlet">
```

Comment:
As far as I know the class loading mechanism and its instantiation of a delayed service component can be delayed until some consumer requests for the service. As `CloudLet`s are service instances (not only bare service components), `AssetCloudlet` has been made a delayed component.